docs/installation/frameworks/vuejs-v3.md

```diff
@@ -342,7 +386,7 @@ module.exports = {
 
 ### Using the editor from source
 
-Having configured `vue.config.js`, you can choose the building blocks of your editor. Install the packages necessary for your integration, but please remember that all packages (excluding `@ckeditor/ckeditor5-dev-*` and `@ckeditor/ckeditor5-vue`) {@link installation/getting-started/installing-plugins#requirements must have the same version as the base editor package}.
+Having your project configured, you can choose the building blocks of your editor. Install the packages necessary for your integration, but please remember that all packages (excluding `@ckeditor/ckeditor5-dev-*` and `@ckeditor/ckeditor5-vue`) {@link installation/getting-started/installing-plugins#requirements must have the same version as the base editor package}.
```

Comment:
`@ckeditor/vite-plugin-ckeditor5` package is also excluded from this requirement, but I'm not sure if this is a good place to mention it.